packages/scripts/preset/README.md

```diff
@@ -0,0 +1,128 @@
+# Preset Talend
+
+This is a preset for @talend/scripts. It holds most of Talend projects tools configuration.
+
+This preset allows some customisation through specific entry points. The configuration is done via `Talend/scripts` configuration file.
```

Comment:
Maybe it wouldn't hurt to repeat the filename somewhere since it's a separate readme: "via `Talend/scripts` configuration file (talend-scripts.json)"? :)

Reply:
Done, added the file name